### PR TITLE
Add codebase for User Page

### PR DESCRIPTION
**Pull request by Andrei-Fabian-Pop**

### CODE REVIEW

File: g9frontend/src/pages/UserPage/UserPageStyle.css [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+    width: 100%;
+    min-height: 0;
+}
+
+.leftContainer {
+    background-color: #a6c8c9;
+    margin-right: 8px;
+    width: 33%;
+    white-space: pre-line;
+}
```
Comment (bia1708) [inline g9frontend/src/pages/UserPage/UserPageStyle.css:61]: The left container does not look like the figma design. It shouldn't be blue.

File: g9frontend/src/pages/UserPage/UserPage.tsx [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+                </div>
+                <div className="content">
+                    <div className="leftContainer">
+                        <div className="welcomeText">
+                            Welcome {username}!
+                        </div>
+                        <div className="centeredContainer">
+                            <Button className="blueButton" style={styles.blueButton}>
+                                Craft your post
+                            </Button>
```
Comment (bia1708) [inline g9frontend/src/pages/UserPage/UserPage.tsx:55]: This button isn't centered and frankly, it's quite ugly. There is also a lot of empty space on the left side.

**Review by bia1708 — CHANGES_REQUESTED** [review]

File: g9frontend/src/pages/UserPage/UserPage.tsx [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+                    <p className="logo">WiFi</p>
+                    <Button style={{
+                        textTransform: 'none',
+                        color: '#f7fefe',
+                        right: '3%',
+                        fontFamily: 'Inter, sans-serif',
+                        fontSize: '20px'
+                    }}>
+                        Log in
+                    </Button>
```
[the remark this answers is not in the document]
Comment (Andrei-Fabian-Pop) [reply g9frontend/src/pages/UserPage/UserPage.tsx:45]: It has no icon, i replaced it with a button that takes the user to the main page

File: g9frontend/src/pages/UserPage/UserPageStyle.css [hunk 4]
```diff
@@ -124,27 +124,27 @@ html, body {
 }
 
 .welcomeText {
-    font-size: 40px; /* Increase the font size */
+    font-size: 80px; /* Increase the font size */
```
Comment (bia1708) [inline g9frontend/src/pages/UserPage/UserPageStyle.css:127]: The welcome text is a bit too big now.

File: g9frontend/src/pages/UserPage/UserPage.tsx [hunk 5]
```diff
@@ -40,8 +41,8 @@ export const UserPage: React.FC = () => {
                         right: '3%',
                         fontFamily: 'Inter, sans-serif',
                         fontSize: '20px'
-                    }}>
-                        Log in
+                    }} to="/" component={Link} color="inherit" sx={{mr: 5}}>
+                        Home Page
```
Comment (bia1708) [inline g9frontend/src/pages/UserPage/UserPage.tsx:45]: Please add this icon to the button: `import PersonOutlineIcon from '@mui/icons-material/PersonOutline';`, like in the Figma design.

File: g9frontend/src/pages/UserPage/UserPageStyle.css [hunk 6]
```diff
@@ -155,7 +155,7 @@ html, body {
     display: flex;
     justify-content: center;
     align-items: center;
-    height: 100vh; /* Optional: Set a height to center vertically within the viewport */
+    height: 60vh; /* Optional: Set a height to center vertically within the viewport */
```
Comment (bia1708) [inline g9frontend/src/pages/UserPage/UserPageStyle.css:158]: You should center it to be (vertically) at the center of the **left container**. Sorry if I wasn't specific enough. It's not exactly centered right now and there is too much space between the text and the 'Craft your post' button.

**Review by bia1708 — CHANGES_REQUESTED** [review]

**Review by bia1708 — APPROVED** [review]

**Pull request merged**